### ISSUE TITLE
Input Gains Focus State Instead of Triggering Display

### ISSUE THREAD

**Issue by spencerfinnell** [qodesmith/datepicker#33]
When using the input to try to trigger the picker on iOS (Safari and Chrome) the input is simply focused instead of displaying the picker.

I have tried adding `readonly` as well.

Possibly related #13

**Comment by Lambo232** [comment]
I can confirm a similar problem.  Works for my app on iphone 7s (ios11) but not on ipad (ios11).   Also a problem on some Android devices eg. works OK on Samsung Galaxy 7 phone but not on Neos tablet.  In my application (popup from icon rather than input element) the datepicker pop's up OK but unable to select either a date or the next/previous month buttons.

**Comment by Lambo232** [comment]
OK, I think I have found the problem.  The datepicker code is written using the "new" javascript ES6 constructs. A lot of mobile devices out there do not support ES6.  You can test for ES6 with the following snip:
```javascript
    var supportsES6 = function() {
        try {
            new Function("(a = 0) => a");
            return true;
        }
        catch (err) {
            return false;
        }
    };
    alert("ES6 Support = "+supportsES6());
```
The only solution I can see is to develop a version using the ES5 standard.

**Comment by qodesmith** [comment]
@Lambo232 This library uses the minified `datepicker.min.js` file, which is all ES5. The issue is something different. I'm looking into it.

**Comment by Lambo232** [comment]
Ah-ha, that explains some of the the issues that I have been having, I assumed the the minified version was ES6 as well.  To try to find my problem I was using the debug version and that is when I had the ES6 issues, damn, I have just spent a couple of hours converting it to ES5 so it would run on the problem devices.

Anyway I have found the "bug" that was causing my problem.  I will submit it as a new issue.
